server/api/products.js

```diff
@@ -29,3 +31,13 @@ router.get('/:id', async (req, res, next) => {
     next(err)
   }
 })
+
+router.post('/admin/add', async (req, res, next) => {
```

Comment:
Might be good to figure out how to get the client-side and api-side routes to match up. But not super-important